app/controllers/classrooms_controller.rb

```diff
@@ -0,0 +1,81 @@
+class ClassroomsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_classroom, only: [:show]
+  before_action :set_classroom, only: [:show, :edit, :update, :destroy, :add_students]
+
+  def index
+    Rails.logger.debug("Current User: #{current_user.inspect}")
+    authorize Classroom
+    @classrooms = policy_scope(Classroom)
+  end
+
+  def show
+    authorize @classroom
+    @participants = @classroom.students
+    # @classroom = find_classroom
+    # authorize @classroom
+
+    # Rails.logger.debug("Participants: #{@classroom.students.inspect}")
+    # @participants = @classroom.students
+
+    # Remove the following line as it's not needed
+    # @classroom = Classroom.find(params[:id])
```

Comment:
Theses commented-out items, do they need to remain? looks like you were trying to solve an issue

Reply:
also this lines fine to remove it it was needed while development right now is not needed anymore